src/misc.cpp

```diff
@@ -39,7 +39,7 @@ using fun3_t = bool(*)(HANDLE, CONST GROUP_AFFINITY*, PGROUP_AFFINITY);
 using fun4_t = bool(*)(USHORT, PGROUP_AFFINITY, USHORT, PUSHORT);
 using fun5_t = WORD(*)();
 using fun6_t = bool(*)(HANDLE, DWORD, PHANDLE);
-using fun7_t = bool(*)(LPCSTR, LPCSTR, PLUID);
+using fun7_t = bool(*)(LPCTSTR, LPCTSTR, PLUID);
```

Comment:
This is incorrect. We explicitly asked for "LookupPrivilegeValueA" which is ANSI.